assets/_sass/_search.scss

```diff
@@ -0,0 +1,36 @@
+// https://github.com/mauriciogentile/angular-livesearch/blob/1f4e357ab3b50701eeedcfb45c12cd901d612087/example/styles.css#L31-L65
+ul.searchresultspopup {
+  border: #a4bed4 1px solid;
+  margin-top: 0px;
+  padding: 0px;
+  z-index: 99999!important;
```

Comment:
Its pretty dangerous to start using z-indexes without organizing them. you could end up having 9999999 next time. I suggest making sass variables for z-indexes.